eng/emitters/pipelines/templates/archetype-emitter.yml

```diff
@@ -69,57 +69,26 @@ extends:
       - stage: Build
         variables:
           selfRepositoryPath: $(Build.SourcesDirectory)
-        pool:
-          name: $(LINUXPOOL)
-          image: $(LINUXVMIMAGE)
-          os: linux
         jobs:
-          - job: Build
+          - job: Linux
```

Comment:
What that issue was more about making sure you are testing on all the node version that are supported so you don't use some new functionality not available yet in still supported node

like here https://github.com/microsoft/typespec/blob/aa95fcbad3f61c52018b13dde4f99994082679b2/eng/tsp-core/pipelines/ci.yml#L36

Reply:
yeah I am working towards that :-)

Reply:
ah ok thought this was ready, let me know

Reply:
I probably should have opened as draft while I am working on it will do that now.